.changeset/honest-states-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-home': minor
```

Comment:
```suggestion
'@backstage/plugin-home': patch
```

As this plugin isn't 1.0 yet this would just be patch, it doesn't look to be breaking either.